codec/block-ethereum.avsc

```diff
@@ -278,6 +278,70 @@
                                        {
                                           "name":"input",
                                           "type":"string"
+                                       },
+                                       {
+                                          "name":"blobFeeCap",
+                                          "type":[
+                                             "null",
+                                             {
+                                                "type":"bytes",
+                                                "logicalType":"decimal",
+                                                "precision":1000
+                                             }
+                                          ],
+                                          "default":"null"
+                                       },
+                                       {
+                                          "name":"blobHashes",
+                                          "type":[
+                                             "null",
+                                             {
+                                                "type":"array",
+                                                "items":"string"
+                                             }
+                                          ],
+                                          "default":"null"
+                                       },
+                                       {
+                                          "name":"blobGas",
+                                          "type":[
+                                             "null",
+                                             "int"
+                                          ],
+                                          "default":"null"
+                                       },
+                                       {
+                                          "name":"blobTxSideCar",
+                                          "type":{
+                                             "name":"blobTxSideCar",
+                                             "type":"record",
+                                             "fields":[
+                                                {
+                                                   "name":"Blobs",
+                                                   "type":[
```

Comment:
i believe you need to change internal/util/utils.go as well to wrap/unwrap these union-type fields.

Reply:
noted good point and made those changes on https://github.com/covalenthq/bsp-agent/pull/245

Reply:
will make them here as well (as we can patch our bsp-geth dencun fork with full eip4844 blob tx support)